docs/userguide/modules/projecting-binding-data/pages/projecting-binding-data.adoc

```diff
@@ -132,6 +132,11 @@ Depending on your workload requirements and environment, you can choose to proje
 ** Configuration of how the final path for data projection is computed for the default method.
 * The binding data is exposed from the backing service.
 
+[NOTE]
+====
+Once the binding is successfully completed, and later the workload is deleted and created again, the projected bindings will be lost.  The ServiceBinding reconciler will not rebind the workload.  This happens when workload is identified through `name`. But if a label selector is used, the rebinding will happen.
```

Comment:
Suggest to put this information as a separate section after the procedure like this:

[#understanding-the-rebinding-behavior]
== Understanding the rebinding behavior
Consider a case where after a successful binding, you are using the `name` field to identify a workload. In such a case, if you delete an existing workload and recreate it, the projected binding data are lost. The `ServiceBinding` reconciler does not rebind the workload. However, if you use the `labelSelector` field to identify a workload, the `ServiceBinding` reconciler rebinds the workload and the Operator projects the binding data.
```

Reply:
Done

Reply:
Thanks, @baijum. Actually I am not sure whether this usage is correct and whether we can write - `ServiceBinding` reconciler. We also have `ServiceBinding` custom resource.

Can you please confirm whether it is ok to write as `ServiceBinding` reconciler? Thanks!

Reply:
@baijum Rest looks good to me. thanks!